Cargo.toml

```diff
@@ -11,7 +11,7 @@ license = "Apache-2.0"
 epoll = ">=4.0.1"
 libc = ">=0.2.39"
 log = ">=0.4.6"
-vhost = { version = "0.1", features = ["vhost-user-slave"] }
+vhost = { git = "https://github.com/harshanavkis/vhost/", rev = "615ee575c6a95001e9a8825d0e743bcd30cced03", features = ["vhost-user-slave"] }
```

Comment:
https://github.com/rust-vmm/vhost/pull/84 is merged, so we can use the https://github.com/rust-vmm/vhost git repository